examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+                      <div class="header">Downloadable formats</div>
+                      <div class="description">PDF, HTML</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Enhanced full-text search</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">API access</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Unlimited projects & users</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Private Versions</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Live PR preview</div>
+                      <div class="description">Live preview of all pull requests</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Email build failure notifications</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">2 day support response time</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">2 Concurrent builds</div>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div class="extra content">
+              <span class="left floated share">
+                <i class="icon share alternate"></i>
+                Share
+              </span>
+              <a class="right floated contact">
+                Learn more
+              </a>
+            </div>
+
+          </div>
+          <!-- end card -->
+          
+          <!-- Card -->
+          <div class="ui inverted card">
+
+            <div class="content">
+              
+              <div class="ui label ribbon orange big">Advanced</div>
+
+              <div class="ui header huge">$150</div>
+              <div class="meta">
+                <span class="category">per month</span>
+              </div>
+
+              <button class="ui button fluid large orange">Subscribe</button>
+
+              <div class="description">
+                
+                <div class="ui list inverted relaxed divided">
+                  <div class="item disabled">
```

Comment:
> I do like the disabled sections for a full grid view however.

Yes. If we end up feeling that this specific style is not the best suited for what we need we can also explore other similar options to maintain visual alignment and information clarity while helping the user skim through repeated information.

> We could consider giving the user this full grid sort of view if they click on a button/etc to expand the grid.

This could be an interesting twist! Not sure how to accomplish it thought, keeping in mind that a pricing page, specially this first one, should be very clear, self explanatory and require minimum action from the user. Maybe we can discuss this further on a call.

> plan highlighting as the primary drivers to users selecting the plan we are trying to promote

Is there one specific plan that we are trying to promote?

Reply:
On commercial, the main plan is probably the Advanced plan. Good question about our community hosting however. We're probably either not highlighting a plan there, or are highlighting the free plan -- our goal isn't necessarily to get users to upgrade to gold or ad free, those are just options we provide users because of advertising.

Reply:
Opened issue #14 to continue the discussion there.